zproject/urls.py

```diff
@@ -169,6 +169,12 @@
     url(r'^messages/flags$', 'zerver.lib.rest.rest_dispatch',
         {'POST': 'zerver.views.messages.update_message_flags'}),
 
+    # typing -> zerver.views.typing
+    # POST sends a typing notification event to recipients
+    # Required parameters are 'op' ('start' or 'stop') and recipients
```

Comment:
We generally don't document parameters in the urls file, since it's the type of thing that gets out of date (and the views function `def` line makes the parameters really clear), so I'd drop the second sentence of this comment.